README.rst

```diff
@@ -232,10 +231,10 @@ Rasterio's Python dependencies are (see the package metadata file):
     matplotlib
 
     [s3]
-    boto3>=1.3.1
+    boto3>=1.2.4
```

Comment:
Let me know if I should instead update `setup.py` to 1.3.1

Reply:
@adamjstewart I'm wondering where you got 1.2.4 from?

Reply:
From the `setup.py`